internal/expr/prepare.go

```diff
@@ -104,11 +104,11 @@ func prepareOutput(ti typeNameToInfo, p *outputPart) ([]fullName, []outputDest,
 		if t.name != "*" {
 			f, ok := info.tagToField[t.name]
 			if !ok {
-				return nil, nil, fmt.Errorf(`type %s has no %q db tag`, info.structType.Name(), t.name)
+				return nil, nil, fmt.Errorf(`type %s has no %q db tag`, info.typ.Name(), t.name)
 			}
 			// For a none star expression we record output destinations here.
```

Comment:
For a non-star expression,

Reply:
Ah I should probably change that somewhere else, I think it doesn't come under the scope of this PR.